src/util/fontawesome/library.js

```diff
@@ -36,6 +36,7 @@ export {
   faSignature,
   faSpaceShuttle,
   faSpinner,
+  faStar,
```

Comment:
we also need to provide the regular version, since this only provides the solid version. Sorry I forgot we don't actually have this icon pack in here yet. without adding it the icon won't load properly.

<img width="256" height="142" alt="Screenshot 2025-09-07 at 11 01 08" src="https://github.com/user-attachments/assets/aefe05ba-36fa-4e4c-8a66-3a5ac3f439f9" />


To fix:

1. run `yarn add "@fortawesome/free-regular-svg-icons@^5.15.4"`
2. add export to `library.js`

```ts
export {
  faStar as farStar,
} from '@fortawesome/free-regular-svg-icons'
```